### DISCUSSION TITLE
Width and Height of Loaded Model

### DISCUSSION THREAD

**Discussion by DubMan1234** [jnsmalm/pixi3d#34] [Q&A]
Good day developer

After I have loaded a model and even if I add it to a view, I am getting the 
.width and .height properties to be zero. Is there anyway I can get the width and height of the model once it has 
been added to a view, or even in the model itself?

Thanks

**Comment by jnsmalm** [comment]
Width and height for a model/mesh is not supported at the moment.

**Comment by jnsmalm** [comment]
Now it's possible to

```
let aabb = mesh.getBoundingBox()
let width = aabb.size.x
let height = aabb.size.y
```
